### ISSUE TITLE
Radio group test plan changes

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-at#1176]
for both plans:
1. Unhide the group label in setup scripts (remove the application of `display: none`).
2. Update commands in relevant tests to reflect the correct number of arrow key presses for all screen readers.
3. Remove group label assertions when moving the screen reader cursor into the start of the radio group with the arrow keys, but not when:
	* navigating into the end of the group;
	* moving focus; or
	* using keys other than the arrows.
4. Update setup script descriptions to remove the references to hiding the group label.

Note: When navigating from a link outside the group to the first radio inside the group, JAWS should require two Down Arrow presses: one for the heading with the group label, and one for the radio.  If the group label is added to the virtual cursor document because of `aria-labelledby` usage, it will require three Down Arrow presses which should result in the test failing.

@IsaDC
